tsl/test/sql/bgw_scheduler_control.sql

```diff
@@ -50,7 +50,12 @@ TRUNCATE _timescaledb_internal.bgw_job_stat;
 -- We change user to make sure that granting SET and ALTER SYSTEM
 -- privileges to the default user actually works.
 --
+-- We test to set it to FATAL and ERROR to make sure that it is
+-- possible, since users might want to disable LOG level as well.
+--
 SET ROLE :ROLE_DEFAULT_PERM_USER;
+ALTER DATABASE :TEST_DBNAME SET timescaledb.bgw_log_level = 'FATAL';
```

Comment:
How do I see that these changes actually apply?

Reply:
I think the only way to test it is creating a C test procedure that emit a FATAL and run it inside a custom job. Everthing using TAP test infrastructure. Don't think normal regression will work because the FATAL will crash the server.

Reply:
> I think the only way to test it is creating a C test procedure that emit a FATAL and run it inside a custom job. 

Yes, it is hard to test.

> [Everything] using TAP test infrastructure. Don't think normal regression will work because the FATAL will crash the server.

Cause the worker to exit with exit code 1, yes, not crash, but I guess that is what you meant.